modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CSharpClientCodegen.java

```diff
@@ -651,13 +651,25 @@ public void postProcessParameter(CodegenParameter parameter) {
         postProcessPattern(parameter.pattern, parameter.vendorExtensions);
         postProcessEmitDefaultValue(parameter.vendorExtensions);
         super.postProcessParameter(parameter);
+        
+        if (nullableType.contains(parameter.dataType)) {
```

Comment:
Should be noted that this code is just copied from the csharp-netcore generator and works properly there